pages/rules.vue

```diff
@@ -7,8 +7,54 @@
 			v-html="$md.render(rules)"
 		/>
 		<div>
-			<p>Je crois qu'il faut être sympa</p>
+			<p>Welcome to the rules page for our event !</p>
+
+			<p>To ensure a fair and enjoyable experience for everyone, please take a moment to read through the competition guidelines.</p>
+
+			<h1> Team </h1>
+			<ul>
+				<li> Each team is limited to a maximum of 4 members. </li>
+				<li> All members must register an account to be eligible for prizes. </li>
+				<li> Each participant can only be part of one team throughout the event. </li>
+				<li> Collaboration with other teams or external parties is strictly prohibited. </li>
+				<li> Teams must choose appropriate names. Offensive or inappropriate names are not allowed. </li>
```

Comment:
`offensive or inappropriate names will be renamed or deleted`

Reply:
done